llama-index-packs/llama-index-packs-finchat/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+classes = ["FinanceChatPack"]
+contains_example = false
+import_path = "llama_index.packs.finchat"
+
+[tool.mypy]
+disallow_untyped_defs = true
+# Remove venv skip when integrated with pre-commit
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Ishan Gupta <345ishaan@gmail.com>"]
+description = "llama-index packs implementation of a hierarchical agent for finance chat."
+keywords = ["agent", "finance", "finchat"]
+license = "MIT"
+maintainers = ["345ishaan"]
+name = "llama-index-packs-finchat"
+packages = [{include = "llama_index/"}]
+readme = "README.md"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<3.12"
+llama-index-core = "^0.10.0"
+tavily-python = "^0.3.1"
```

Comment:
we are missing a few dependencies here
`llama-index-agents-openai`
`llama-index-tools-finance`